python/lsst/ts/scheduler/scheduler_csc.py

```diff
@@ -492,8 +533,42 @@ async def do_load(self, data):
                 "Target production loop is running. Stop it before loading a file."
             )
 
+        await self.cmd_load.ack_in_progress(
+            data,
+            timeout=self.default_command_timeout,
+            result="Loading snapshot.",
+        )
+
         await self._handle_load_snapshot(data.uri)
 
+    async def _do_computePredictedSchedule(self, data):
+        """Compute and publish the predicted schedule.
+
+        This command can only be executed if the Scheduler is idle. It it is
+        currently running, the command will be rejected.
+
+        Parameters
+        ----------
+        data : `DataType`
+            Command data.
+        """
+
+        self.assert_enabled()
+
+        if self.run_target_loop.is_set():
+            raise RuntimeError(
+                "Scheduler is currently running. Note that the predicted schedule "
+                "is computed automatically while the Scheduler is running."
```

Comment:
Perhaps I misunderstood but isn't the predicted schedule computed while the Scheduler is NOT running?

Reply:
No, when the Scheduler is running it computes the predicted queue and publishes the information. But this command allows one to request the Scheduler to compute and publish the information when it is NOT running. 

So, for instance, if we stopped the Scheduler for some engineering and we want to see what is coming up next, while we finish the engineering task, we can send this command.